syncer/checkpoint.go

```diff
@@ -145,10 +145,11 @@ type CheckPoint interface {
 	// corresponding to Meta.Save
 	SaveGlobalPoint(pos mysql.Position)
 
-	// FlushGlobalPointsExcept flushes the global checkpoint and tables' checkpoints except exceptTables
+	// FlushGlobalPointsExcept flushes the global checkpoint and tables'
+	// checkpoints except exceptTables, it also flushes sharding meta infomation
 	// @exceptTables: [[schema, table]... ]
 	// corresponding to Meta.Flush
-	FlushPointsExcept(exceptTables [][]string) error
+	FlushPointsExcept(exceptTables [][]string, shardMetaSQLs []string, shardMetaArgs [][]interface{}) error
```

Comment:
rename `shardMetaSQLs`/`shardMetaArgs` to other common names? maybe we will execute more SQLs combine with the checkpoint in the same transaction.